docs/workflow/testing/libraries/testing-apple.md

```diff
@@ -1,34 +1,89 @@
-# Testing Libraries on iOS and tvOS
+# Testing Libraries on iOS, tvOS, and MacCatalyst
 
-In order to build libraries and tests for iOS or tvOS you need recent version of XCode installed (e.g. 11.3 or higher).
+## Prerequisites
 
-Build Libraries for iOS Simulator:
+- XCode 11.3 or higher
+
+## Building Libs and Tests
+
+You can build and run the library tests:
+- on a simulator;
+- on a device.
+
+Run the following command in a terminal:
+```
+./build.sh mono+libs -os <TARGET_OS> -arch x64
+```
+where `<TARGET_OS>` is one of the following:
+
+| simulator     | device|
+|:--------------|:------|
+| iOSSimulator  | iOS   |
+| tvOSSimulator | tvOS  |
+| MacCatalyst   |       |
```

Comment:
The simulator vs. device concept doesn't apply for MacCatalyst since it's just a "normal" app running on macOS. I don't think the table provides a lot of value here, I'd just use a bullet-point list.